### ISSUE TITLE
Fields won't autowire if managed bean has invalid name (not in camelCase)

### ISSUE THREAD

**Issue by stochmalm** [joinfaces/joinfaces#432]
I'm not sure if it's a joinfaces issue, but autowired fields in my managed bean won't get injected if my managed bean has wrong name (not in camelCase) .

This works:
```
@ManagedBean
@ViewScoped
@Component
@Log4j2
public class UcarsContractsMB {

	@Autowired
	private ContractService contractService;

	@Autowired
	private DictionaryService dictionaryService;
...
}
```
This doesn't:
```
@ManagedBean
@ViewScoped
@Component
@Log4j2
public class UCarsContractsMB {

	@Autowired
	private ContractService contractService; // contractService == null

	@Autowired
	private DictionaryService dictionaryService; // dictionaryService == null
...
}
```

**Comment by larsgrefer** [comment]
**TL;DR**: Try to remove the `@ManagedBean`-annotation. You don't need this with Joinfaces.

Since your class is Annotated with `@ManagedBean` and `@Component` you actually get two beans of type `UCarsContractsMB`. One managed by JSF and one managed by Spring.
The `@Autowired`-annotation only works for beans managed by Spring.

**Comment by stochmalm** [comment]
Ok, thanks! :)

**Comment by larsgrefer** [comment]
Joinfaces(Spring) completely ignores the `@ManagedBean` annotation, but JSF (Mojarra/MyFaces) does not.

**Comment by stochmalm** [comment]
Yes, thanks for help.
